lib/src/peer.dart

```diff
@@ -39,13 +39,17 @@ class Peer implements Client, Server {
   Future get done => _manager.done;
   bool get isClosed => _manager.isClosed;
 
+  @override
+  ErrorCallback get onUnhandledError => _server?.onUnhandledError;
+
   /// Creates a [Peer] that communicates over [channel].
   ///
   /// Note that the peer won't begin listening to [channel] until [Peer.listen]
   /// is called.
-  Peer(StreamChannel<String> channel)
+  Peer(StreamChannel<String> channel, {ErrorCallback onUnhandledError})
```

Comment:
This argument should be called out in the doc comment.

Reply:
Done